lib/commands.py

```diff
@@ -171,17 +172,39 @@ def commands(self):
         return ' '.join(sorted(known_commands.keys()))
 
     @command('')
-    def create(self):
-        """Create a new wallet"""
-        raise BaseException('Not a JSON-RPC command')
+    def create(self, passphrase=None, password=None, encrypt_file=True, seed_type=None, wallet_path=None):
```

Comment:
I am getting the following error when running this branch of electron-cash:

```
Traceback (most recent call last):
  File "./electron-cash", line 351, in <module>
    parser = get_parser()
  File "/Users/calin/src/ec/electron-cash/lib/commands.py", line 961, in get_parser
    a, help = command_options[optname]
KeyError: 'passphrase'
```

You need to define each of these arguments that are new in the `param_descriptions` dict and `command_options` dicts in `commands.py`.

New arguments seem to be:

- `passphrase`
- `encrypt_file`
- `seed_type`

All of the above need entries in the above two dicts.

Reply:
Added that, now it works
![image](https://user-images.githubusercontent.com/39452697/66266029-02833580-e828-11e9-8d3f-8a18faf181d1.png)